src/main/java/com/descope/sdk/auth/MagicLinkService.java

```diff
@@ -19,7 +18,7 @@ public interface MagicLinkService {
    * @param deliveryMethod - {@link DeliveryMethod DeliveryMethod}
    * @param loginId - User login ID
    * @param uri - Base URI
-   * @param request - {@link HttpRequest HttpRequest}
+   * @param token - when doing step-up or mfa then we need current session token
```

Comment:
This is breaking compatibility no ?

Reply:
Don't believe anyone used it because it was not very usable the way it was. This type of request is not what you get from server requests in web frameworks.